contracts/stable_coin_factory/sources/tests/kasa_manager/liquidation.move

```diff
@@ -25,6 +25,10 @@ module stable_coin_factory::kasa_manager_liquidation_tests {
 
         init_stable_coin_factory(test);
 
+        next_tx(test, admin);
+        {
+            svalue_feed_holder::create_oracle_holder_for_test(test::ctx(test));
```

Comment:
This can be called in `init_stable_coin_factory` method since we need to initialize this for every test. Better to use the already existing code.